### ISSUE TITLE
Relative screenshot path does not work for custom specPattern

### ISSUE THREAD

**Issue by sebkasanzew** [cypress-visual-regression/cypress-visual-regression#152]
I have set a custom [`specPattern`](https://docs.cypress.io/guides/references/configuration#e2e) for my visual regression tests:
```
cypress open --config specPattern=\"**/visual/**/*.cy.ts\"
```

Now because [this code](https://github.com/cypress-visual-regression/cypress-visual-regression/blob/c7714f240d38a95cc2a95c7db97a7154c70393a2/src/command.js#L25) only strips away the default spec pattern, my `base` and `diff` folders look really weird:

base: `cypress/snapshots/visual/base/cypress/visual/`
diff: `cypress/snapshots/visual/diff/cypress/visual/`

So I need to set the env variable `INTEGRATION_FOLDER` to `cypress/visual/`, so the path is created correctly.

**Does anyone have a better idea?**

I guess it would be really hard to parse the regex of `specPattern` in order to get the right string for stripping the path, right?

**Comment by Zaista** [comment]
Don't think so, for the moment, that's the correct way to fix the issue
